content/docs/introducing-jsx.md

```diff
@@ -98,7 +98,7 @@ const element = <div tabIndex="0"></div>;
 You may also use curly braces to embed a JavaScript expression in an attribute:
 
 ```js
-const element = <img src={user.avatarUrl}></img>;
+const element = <img src={user.avatarUrl} alt='introducing-jsx'></img>;
```

Comment:
The `alt` would be better, if it related with the img content, so:
```suggestion
const element = <img src={user.avatarUrl} alt='avatar-url'></img>;
```

Reply:
I re-read the doc and i think you are right. 
`alt`  attribute specifies an alternate text for an image.
I have tried VoiceOver and i think code `src={user.avatarUrl}` is clear enough for everyone, so i removed value of `alt` prop.

Reply:
Yeah, but the `alt` attribute is a alternative text, when image is not rendered, so it's still cool keep the attribute.